forge-gui/res/languages/en-US.properties

```diff
@@ -720,11 +720,15 @@ lblInvalidDeck=Invalid Deck
 lblInvalidDeckDesc=Your deck %n\nPlease edit or choose a different deck.
 #VSubmenuQuestPrefs.java
 lblQuestPreferences=Quest Preferences
+lblQuestGameSettingsError=Game Settings Error
```

Comment:
please add localization for other languages even if it's english to prevent error text displaying on labels

Reply:
Already did in the previous MR. No further changes to the localization here.